Zammad fails at Elasticsearch

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

Zammad

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/zammad.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📈 Which Proxmox version are you on?

pve-manager/9.1.5/80cf92a64bef6889 (running kernel: 6.17.9-1-pve)

### 📝 Provide a clear and concise description of the issue.

[ERROR] in line 35: exit code 1 (General error / Operation not permitted): while executing command systemctl restart -q elasticsearch

### 🔄 Steps to reproduce the issue.

Run script

Go through prompts

### ❌ Paste the full error output (if available).
```
🧩  Using Advanced Install on node pve

  💡  PVE Version 9.1.5 (Kernel: 6.17.9-1-pve)
  🖥️  Operating System: debian
  🌟  Version: 12
  📦  Container Type: Unprivileged
  🆔  Container ID: 101
  🏠  Hostname: zammad
  💾  Disk Size: 8 GB
  🧠  CPU Cores: 2
  🛠️  RAM Size: 4096 MiB
  🌉  Bridge: vmbr0
  📡  IPv4: 10.0.0.204/23
  📡  IPv6: auto
  🗂️  FUSE Support: no
  📡  TUN/TAP Support: yes
  📦  Nesting: Enabled
  🎮  GPU Passthrough: no
  💡  Timezone: America/Denver
  🔍  Verbose Mode: yes
  🚀  Creating an LXC of Zammad using the above advanced settings
 ℹ️ Keeping current app defaults: /usr/local/community-scripts/defaults/zammad.vars
  ✔️   Storage local (Free: 84.2GB  Used: 5.0GB) [Template]
  ✔️   Storage local-lvm (Free: 1.7TB  Used: 700.4MB) [Container]
  ✔️   Storage 'local-lvm' (lvmthin) validated
  ✔️   Template storage 'local' validated
  ✔️   Template search completed
  ✔️   Template debian-12-standard_12.12-1_amd64.tar.zst [local]
  ✔️   LXC Container 101 was successfully created.
  ✔️   Started LXC Container
  ✔️   Network in LXC is reachable (ping)
  ✔️   Customized LXC Container
  ✔️   Set up Container OS
  ✔️   Network Connected: 10.0.0.204 
  ✔️   IPv4 Internet Connected
  ✖️   IPv6 Internet Not Connected
  ✔️   Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 http://security.debian.org bookworm-security InRelease
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
  base-files bash bind9-dnsutils bind9-host bind9-libs distro-info-data inetutils-telnet libcap2 libcap2-bin libgnutls30 libpam-modules libpam-modules-bin
  libpam-runtime libpam0g libssl3 libxml2 login openssl passwd python3-urllib3
20 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 11.0 MB of archives.
After this operation, 32.8 kB of additional disk space will be used.
Get:1 http://security.debian.org bookworm-security/main amd64 libgnutls30 amd64 3.7.9-2+deb12u6 [1,408 kB]
Get:2 http://deb.debian.org/debian bookworm/main amd64 base-files amd64 12.4+deb12u13 [70.9 kB]
Get:3 http://deb.debian.org/debian bookworm/main amd64 bash amd64 5.2.15-2+b10 [1,490 kB]
Get:4 http://security.debian.org bookworm-security/main amd64 libssl3 amd64 3.0.18-1~deb12u2 [2,030 kB]
Get:5 http://security.debian.org bookworm-security/main amd64 bind9-dnsutils amd64 1:9.18.44-1~deb12u1 [155 kB]
Get:6 http://security.debian.org bookworm-security/main amd64 bind9-host amd64 1:9.18.44-1~deb12u1 [54.7 kB]
Get:7 http://security.debian.org bookworm-security/main amd64 bind9-libs amd64 1:9.18.44-1~deb12u1 [1,183 kB]
Get:8 http://security.debian.org bookworm-security/main amd64 inetutils-telnet amd64 2:2.4-2+deb12u2 [120 kB]
Get:9 http://security.debian.org bookworm-security/main amd64 openssl amd64 3.0.18-1~deb12u2 [1,433 kB]
Get:10 http://deb.debian.org/debian bookworm/main amd64 login amd64 1:4.13+dfsg1-1+deb12u2 [616 kB]
Get:11 http://security.debian.org bookworm-security/main amd64 python3-urllib3 all 1.26.12-1+deb12u3 [114 kB]
Get:12 http://deb.debian.org/debian bookworm/main amd64 libpam0g amd64 1.5.2-6+deb12u2 [92.1 kB]
Get:13 http://deb.debian.org/debian bookworm/main amd64 libpam-modules-bin amd64 1.5.2-6+deb12u2 [75.7 kB]
Get:14 http://deb.debian.org/debian bookworm/main amd64 libpam-modules amd64 1.5.2-6+deb12u2 [291 kB]
Get:15 http://deb.debian.org/debian bookworm/main amd64 libpam-runtime all 1.5.2-6+deb12u2 [161 kB]
Get:16 http://deb.debian.org/debian bookworm/main amd64 passwd amd64 1:4.13+dfsg1-1+deb12u2 [972 kB]
Get:17 http://deb.debian.org/debian bookworm/main amd64 libcap2 amd64 1:2.66-4+deb12u2+b2 [27.5 kB]
Get:18 http://deb.debian.org/debian bookworm/main amd64 libxml2 amd64 2.9.14+dfsg-1.3~deb12u5 [688 kB]
Get:19 http://deb.debian.org/debian bookworm/main amd64 distro-info-data all 0.58+deb12u6 [6,528 B]
Get:20 http://deb.debian.org/debian bookworm/main amd64 libcap2-bin amd64 1:2.66-4+deb12u2+b2 [35.2 kB]
Fetched 11.0 MB in 1s (9,540 kB/s)      
apt-listchanges: Reading changelogs...
Preconfiguring packages ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../base-files_12.4+deb12u13_amd64.deb ...
Unpacking base-files (12.4+deb12u13) over (12.4+deb12u12) ...
Setting up base-files (12.4+deb12u13) ...
Installing new version of config file /etc/debian_version ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../bash_5.2.15-2+b10_amd64.deb ...
Unpacking bash (5.2.15-2+b10) over (5.2.15-2+b9) ...
Setting up bash (5.2.15-2+b10) ...
update-alternatives: using /usr/share/man/man7/bash-builtins.7.gz to provide /usr/share/man/man7/builtins.7.gz (builtins.7.gz) in auto mode
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../login_1%3a4.13+dfsg1-1+deb12u2_amd64.deb ...
Unpacking login (1:4.13+dfsg1-1+deb12u2) over (1:4.13+dfsg1-1+deb12u1) ...
Setting up login (1:4.13+dfsg1-1+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libpam0g_1.5.2-6+deb12u2_amd64.deb ...
Unpacking libpam0g:amd64 (1.5.2-6+deb12u2) over (1.5.2-6+deb12u1) ...
Setting up libpam0g:amd64 (1.5.2-6+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libpam-modules-bin_1.5.2-6+deb12u2_amd64.deb ...
Unpacking libpam-modules-bin (1.5.2-6+deb12u2) over (1.5.2-6+deb12u1) ...
Setting up libpam-modules-bin (1.5.2-6+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libpam-modules_1.5.2-6+deb12u2_amd64.deb ...
Unpacking libpam-modules:amd64 (1.5.2-6+deb12u2) over (1.5.2-6+deb12u1) ...
Setting up libpam-modules:amd64 (1.5.2-6+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libpam-runtime_1.5.2-6+deb12u2_all.deb ...
Unpacking libpam-runtime (1.5.2-6+deb12u2) over (1.5.2-6+deb12u1) ...
Setting up libpam-runtime (1.5.2-6+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../passwd_1%3a4.13+dfsg1-1+deb12u2_amd64.deb ...
Unpacking passwd (1:4.13+dfsg1-1+deb12u2) over (1:4.13+dfsg1-1+deb12u1) ...
Setting up passwd (1:4.13+dfsg1-1+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libcap2_1%3a2.66-4+deb12u2+b2_amd64.deb ...
Unpacking libcap2:amd64 (1:2.66-4+deb12u2+b2) over (1:2.66-4+deb12u2) ...
Setting up libcap2:amd64 (1:2.66-4+deb12u2+b2) ...
(Reading database ... 20347 files and directories currently installed.)
Preparing to unpack .../libgnutls30_3.7.9-2+deb12u6_amd64.deb ...
Unpacking libgnutls30:amd64 (3.7.9-2+deb12u6) over (3.7.9-2+deb12u5) ...
Setting up libgnutls30:amd64 (3.7.9-2+deb12u6) ...
(Reading database ... 20347 files and directories currently installed.)
Preparing to unpack .../0-libssl3_3.0.18-1~deb12u2_amd64.deb ...
Unpacking libssl3:amd64 (3.0.18-1~deb12u2) over (3.0.17-1~deb12u2) ...
Preparing to unpack .../1-libxml2_2.9.14+dfsg-1.3~deb12u5_amd64.deb ...
Unpacking libxml2:amd64 (2.9.14+dfsg-1.3~deb12u5) over (2.9.14+dfsg-1.3~deb12u4) ...
Preparing to unpack .../2-bind9-dnsutils_1%3a9.18.44-1~deb12u1_amd64.deb ...
Unpacking bind9-dnsutils (1:9.18.44-1~deb12u1) over (1:9.18.33-1~deb12u2) ...
Preparing to unpack .../3-bind9-host_1%3a9.18.44-1~deb12u1_amd64.deb ...
Unpacking bind9-host (1:9.18.44-1~deb12u1) over (1:9.18.33-1~deb12u2) ...
Preparing to unpack .../4-bind9-libs_1%3a9.18.44-1~deb12u1_amd64.deb ...
Unpacking bind9-libs:amd64 (1:9.18.44-1~deb12u1) over (1:9.18.33-1~deb12u2) ...
Preparing to unpack .../5-inetutils-telnet_2%3a2.4-2+deb12u2_amd64.deb ...
Unpacking inetutils-telnet (2:2.4-2+deb12u2) over (2:2.4-2+deb12u1) ...
Preparing to unpack .../6-distro-info-data_0.58+deb12u6_all.deb ...
Unpacking distro-info-data (0.58+deb12u6) over (0.58+deb12u5) ...
Preparing to unpack .../7-libcap2-bin_1%3a2.66-4+deb12u2+b2_amd64.deb ...
Unpacking libcap2-bin (1:2.66-4+deb12u2+b2) over (1:2.66-4+deb12u2) ...
Preparing to unpack .../8-openssl_3.0.18-1~deb12u2_amd64.deb ...
Unpacking openssl (3.0.18-1~deb12u2) over (3.0.17-1~deb12u2) ...
Preparing to unpack .../9-python3-urllib3_1.26.12-1+deb12u3_all.deb ...
Unpacking python3-urllib3 (1.26.12-1+deb12u3) over (1.26.12-1+deb12u1) ...
Setting up distro-info-data (0.58+deb12u6) ...
Setting up libssl3:amd64 (3.0.18-1~deb12u2) ...
Setting up inetutils-telnet (2:2.4-2+deb12u2) ...
Setting up libcap2-bin (1:2.66-4+deb12u2+b2) ...
Setting up python3-urllib3 (1.26.12-1+deb12u3) ...
Setting up openssl (3.0.18-1~deb12u2) ...
Setting up libxml2:amd64 (2.9.14+dfsg-1.3~deb12u5) ...
Setting up bind9-libs:amd64 (1:9.18.44-1~deb12u1) ...
Setting up bind9-host (1:9.18.44-1~deb12u1) ...
Setting up bind9-dnsutils (1:9.18.44-1~deb12u1) ...
Processing triggers for man-db (2.11.2-2) ...
Processing triggers for debianutils (5.7-0.5~deb12u1) ...
Processing triggers for libc-bin (2.36-9+deb12u13) ...
  ✔️   Updated Container OS
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  git-man liberror-perl nginx-common patch
Suggested packages:
  git-daemon-run | git-daemon-sysvinit git-doc git-email git-gui gitk gitweb git-cvs git-mediawiki git-svn fcgiwrap nginx-doc ed diffutils-doc
The following NEW packages will be installed:
  apt-transport-https git git-man liberror-perl nginx nginx-common patch
0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.
Need to get 10.1 MB of archives.
After this operation, 50.2 MB of additional disk space will be used.
Get:1 http://security.debian.org bookworm-security/main amd64 nginx-common all 1.22.1-9+deb12u4 [113 kB]
Get:2 http://security.debian.org bookworm-security/main amd64 nginx amd64 1.22.1-9+deb12u4 [528 kB]
Get:3 http://deb.debian.org/debian bookworm/main amd64 apt-transport-https all 2.6.1 [25.2 kB]
Get:4 http://deb.debian.org/debian bookworm/main amd64 liberror-perl all 0.17029-2 [29.0 kB]
Get:5 http://deb.debian.org/debian bookworm/main amd64 git-man all 1:2.39.5-0+deb12u3 [2,053 kB]
Get:6 http://deb.debian.org/debian bookworm/main amd64 git amd64 1:2.39.5-0+deb12u3 [7,264 kB]
Get:7 http://deb.debian.org/debian bookworm/main amd64 patch amd64 2.7.6-7 [128 kB]
Fetched 10.1 MB in 1s (11.3 MB/s)
Preconfiguring packages ...
Selecting previously unselected package apt-transport-https.
(Reading database ... 20351 files and directories currently installed.)
Preparing to unpack .../0-apt-transport-https_2.6.1_all.deb ...
Unpacking apt-transport-https (2.6.1) ...
Selecting previously unselected package liberror-perl.
Preparing to unpack .../1-liberror-perl_0.17029-2_all.deb ...
Unpacking liberror-perl (0.17029-2) ...
Selecting previously unselected package git-man.
Preparing to unpack .../2-git-man_1%3a2.39.5-0+deb12u3_all.deb ...
Unpacking git-man (1:2.39.5-0+deb12u3) ...
Selecting previously unselected package git.
Preparing to unpack .../3-git_1%3a2.39.5-0+deb12u3_amd64.deb ...
Unpacking git (1:2.39.5-0+deb12u3) ...
Selecting previously unselected package nginx-common.
Preparing to unpack .../4-nginx-common_1.22.1-9+deb12u4_all.deb ...
Unpacking nginx-common (1.22.1-9+deb12u4) ...
Selecting previously unselected package nginx.
Preparing to unpack .../5-nginx_1.22.1-9+deb12u4_amd64.deb ...
Unpacking nginx (1.22.1-9+deb12u4) ...
Selecting previously unselected package patch.
Preparing to unpack .../6-patch_2.7.6-7_amd64.deb ...
Unpacking patch (2.7.6-7) ...
Setting up apt-transport-https (2.6.1) ...
Setting up nginx-common (1.22.1-9+deb12u4) ...
Created symlink /etc/systemd/system/multi-user.target.wants/nginx.service → /lib/systemd/system/nginx.service.
Setting up liberror-perl (0.17029-2) ...
Setting up patch (2.7.6-7) ...
Setting up git-man (1:2.39.5-0+deb12u3) ...
Setting up nginx (1.22.1-9+deb12u4) ...
Upgrading binary: nginx.
Setting up git (1:2.39.5-0+deb12u3) ...
Processing triggers for man-db (2.11.2-2) ...
  ✔️   Installed Dependencies
Get:1 https://artifacts.elastic.co/packages/7.x/apt stable InRelease [13.7 kB]
Hit:2 http://deb.debian.org/debian bookworm InRelease                   
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Hit:4 http://security.debian.org bookworm-security InRelease
Get:5 https://artifacts.elastic.co/packages/7.x/apt stable/main amd64 Packages [145 kB]
Fetched 158 kB in 1s (145 kB/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following NEW packages will be installed:
  elasticsearch
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 325 MB of archives.
After this operation, 542 MB of additional disk space will be used.
Get:1 https://artifacts.elastic.co/packages/7.x/apt stable/main amd64 elasticsearch amd64 7.17.29 [325 MB]
Fetched 325 MB in 12s (26.5 MB/s)                                                                                                                            
Selecting previously unselected package elasticsearch.
(Reading database ... 21528 files and directories currently installed.)
Preparing to unpack .../elasticsearch_7.17.29_amd64.deb ...
Creating elasticsearch group... OK
Creating elasticsearch user... OK
Unpacking elasticsearch (7.17.29) ...
Setting up elasticsearch (7.17.29) ...
### NOT starting on installation, please execute the following statements to configure elasticsearch service to start automatically using systemd
 sudo systemctl daemon-reload
 sudo systemctl enable elasticsearch.service
### You can start elasticsearch service by executing
 sudo systemctl start elasticsearch.service
Created elasticsearch keystore in /etc/elasticsearch/elasticsearch.keystore
-> Installing ingest-attachment
-> Downloading ingest-attachment from elastic
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@     WARNING: plugin requires additional permissions     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
* java.lang.RuntimePermission accessClassInPackage.sun.java2d.cmm.kcms
* java.lang.RuntimePermission accessDeclaredMembers
* java.lang.RuntimePermission getClassLoader
* java.lang.reflect.ReflectPermission suppressAccessChecks
* java.security.SecurityPermission createAccessControlContext
See https://docs.oracle.com/javase/8/docs/technotes/guides/security/permissions.html
for descriptions of what these permissions allow and the associated risks.
-> Installed ingest-attachment
-> Please restart Elasticsearch to activate any plugins installed
  ✖️   in line 35: exit code 1 (General error / Operation not permitted): while executing command systemctl restart -q elasticsearch

  --- Last 20 lines of log ---
[2026-02-20 07:56:51] [INFO] Setting up Container OS
[2026-02-20 07:56:51] [OK] Set up Container OS
[2026-02-20 07:56:51] [OK] Network Connected: 10.0.0.204 
[2026-02-20 07:56:52] [OK] IPv4 Internet Connected
[2026-02-20 07:56:52] [ERROR] IPv6 Internet Not Connected
[2026-02-20 07:56:53] [OK] Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
[2026-02-20 07:56:53] [INFO] Updating Container OS
[2026-02-20 07:57:24] [OK] Updated Container OS
[2026-02-20 07:57:25] [INFO] Installing Dependencies
[2026-02-20 07:57:38] [OK] Installed Dependencies
[2026-02-20 07:57:38] [INFO] Setting up Elasticsearch
[2026-02-20 07:58:49] [ERROR] in line 35: exit code 1 (General error / Operation not permitted): while executing command systemctl restart -q elasticsearch
  -----------------------------------

  ✖️   Installation failed in container 101 (exit code: 1)

[1]+  Stopped                 bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/zammad.sh)"
```
### 🖼️ Additional context (optional).

Used Advanced Install to set static IP and turn on TUN/TAP Support for Tailscale.
Everything else was left as default.

I have tried with Default Install and met with the same error.

--- a/install/zammad-install.sh
+++ b/install/zammad-install.sh
@@ -28,9 +28,10 @@ setup_deb822_repo \
   "stable" \
   "main"
 $STD apt install -y elasticsearch
-echo "-Xms2g" >>/etc/elasticsearch/jvm.options
-echo "-Xmx2g" >>/etc/elasticsearch/jvm.options
+sed -i 's/^-Xms.*/-Xms2g/' /etc/elasticsearch/jvm.options
+sed -i 's/^-Xmx.*/-Xmx2g/' /etc/elasticsearch/jvm.options
 $STD /usr/share/elasticsearch/bin/elasticsearch-plugin install ingest-attachment -b
+systemctl daemon-reload
 systemctl enable -q elasticsearch
 systemctl restart -q elasticsearch
 msg_ok "Setup Elasticsearch"

